os/fs/smartfs/README.txt

```diff
@@ -35,7 +35,7 @@ Features
     - Selectable FLASH Wear leveling algorithym
     - Selectable CRC-8 or CRC-16 error detection for sector data
     - Reduced RAM model for FLASH geometries with large number of sectors (16K-64K)
-    - Recovery corrupted sector & Logging based Journaling
+    - Recovery corrupted sector & Logging based Journaling //TODO: Re-implement journaling
```

Comment:
TODO is not necessary, we will merge to master branch when we implemented basic operation.

Reply:
Done